### PR TITLE
Add flow stream doc

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: documentation/api.md [hunk 1]
```diff
@@ -139,3 +141,57 @@ Content-Type: application/json
 HTTP/1.1 200 OK
 Content-Type: application/json; charset=UTF-8
 {% endhighlight %}
+
+## WebSocket API
+
+### Flow stream
+
+Since the version `0.21` it is possible to subscribe to analyzers flow stream. This feature can be use
```
Comment (lebauce) [inline documentation/api.md]: Since version `0.21`

File: documentation/api.md [hunk 2]
```diff
@@ -139,3 +141,57 @@ Content-Type: application/json
...
+to react on new flow event or to store/send them to an external tool. The flow stream is provided as
```
Comment (lebauce) [inline documentation/api.md]: new flows events

File: documentation/api.md [hunk 3]
```diff
@@ -139,3 +141,57 @@ Content-Type: application/json
...
+a WebSocket endpoint. You can use the `skydive-client` Python library or the Golang one. By default
+flows will be sent in JSON but can be sent in Protobuf if requested.
```
Comment (lebauce) [inline documentation/api.md]: , but can also be sent

File: documentation/api.md [hunk 4]
```diff
@@ -139,3 +141,57 @@ Content-Type: application/json
...
+
+The WebSocket endpoint is `/ws/subscriber/flow`
+
+The following example shows how to subscribe with the Python library and to write the flows in a plain file.
```
Comment (lebauce) [inline documentation/api.md]: using the Python library, and write the received flows to a file.

File: documentation/api.md [hunk 5]
```diff
@@ -139,3 +141,57 @@ Content-Type: application/json
...
 {% endhighlight %}
+
+## WebSocket API
+
+### Flow stream
+
+Since version `0.21` it is possible to subscribe to analyzers flow stream. This feature can be use
+to react on new flow events or to store/send them to an external tool. The flow stream is provided as
+a WebSocket endpoint. You can use the `skydive-client` Python library or the Golang one. By default
+flows, but can also be sent in JSON but can be sent in Protobuf if requested.
```
Comment (nplanel) [inline documentation/api.md]: By default flow are sent in JSON format, but ....

**Pull request merged**